### PR TITLE
Feature/add missing fields and columns to team overview

### PR DESCRIPTION
**Pull request by nerdsupremacist**

### CODE REVIEW

File: src/server/views/partials/admin-teamoverview.handlebars [hunk 1]
```diff
@@ -29,10 +29,11 @@
...
             <thead>
             <tr>
-                <th>Team-Name</th>
-                <th>Mitglieder</th>
+                <th>Status</th>
+                <th style="width: 150px">Team</th>
+                <th style="width: 200px">Participants</th>
                 <th>
-                  Letzer Post
+                  Last Post
```
Comment (thepiwo) [inline src/server/views/partials/admin-teamoverview.handlebars:36]: No Translations needed for this?

**Review by thepiwo — APPROVED** [review]
Would have been cool to build with react, but looks fine like this

**Comment by nerdsupremacist** [comment]
We can still build it with react later 😉

File: src/server/views/partials/admin-teamoverview.handlebars (hunk 1 above)
Comment (nerdsupremacist) [reply src/server/views/partials/admin-teamoverview.handlebars:36]: Got lazy. And this is only for admins anyway, where nothing is translated

**Review by thepiwo — APPROVED** [review]

**Pull request merged**